### ISSUE TITLE
Array of T is translated into T?

### ISSUE THREAD

**Issue by StefH** [paulirwin/JavaToCSharp#38]
# java
``` java
public class Big5Statistics extends nsEUCStatistics {

     static float[] mFirstByteFreq ;

     public Big5Statistics() {

         mFirstByteFreq = new float[] { 
                    0.000000f, ...
```

# c#
``` c#
public class Big5Statistics : nsEUCStatistics
{
	static float[] mFirstByteFreq;

	public Big5Statistics()
	{
		mFirstByteFreq = new float {0F, ...
```

It should be:
# c# fixed
``` c#
public class Big5Statistics : nsEUCStatistics
{
	static float[] mFirstByteFreq;

	public Big5Statistics()
	{
		mFirstByteFreq = new float[] {0F, ... 
```
```

**Comment by StefH** [comment]
![image](https://github.com/paulirwin/JavaToCSharp/assets/249938/ccb2d470-d232-4532-9c47-2c22ba267e1b)

**Comment by paulirwin** [comment]
Fixed in trunk for the upcoming 3.0 release.
